app/javascript/components/shared/Header.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import { Navbar, NavDropdown, Stack } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { useAuth } from '../sessions/AuthContext';
+
+export default function Header() {
+  const currentUser = useAuth();
+
+  return (
+    <Navbar bg="light">
+      <Container>
+        <Navbar.Brand as={Link} to="/">
+          <img
+            src="https://blindsidenetworks.com/wp-content/uploads/2021/04/cropped-bn_logo-02.png"
+            width="200"
+            height=""
+            className="d-inline-block align-top"
+            alt="CompanyLogo"
+          />
+        </Navbar.Brand>
+
+        {currentUser?.signed_in
+          ? (
+            <NavDropdown title={currentUser?.name} id="basic-nav-dropdown">
+              <NavDropdown.Item as={Link} to="/">Profile</NavDropdown.Item>
+              <NavDropdown.Item as={Link} to="/">Need help?</NavDropdown.Item>
+              <NavDropdown.Divider />
+              <NavDropdown.Item as={Link} to="/" onClick={() => console.log("signOut")}>Sign Out</NavDropdown.Item>
```

Comment:
Maybe change this to the actual sign_out request once the sign out PR is done

Reply:
Yep, will do